lte/gateway/python/magma/pipelined/app/ipv6_solicitation.py

```diff
@@ -90,11 +91,15 @@ def _get_config(self, config_dict):
             ipv6_src = get_link_local_ipv6_from_if(s1_interface)
             if ipv6_src is None:
                 ipv6_src = self.DEFAULT_LINK_LOCAL_ADDR
-
+        ng_service = config_dict.get('enable5g_features', None)
+        logging.info("enable5g")
+        logging.info(ng_service)
         return self.IPv6RouterConfig(
             ipv6_src=ipv6_src,
             ll_addr=config_dict['virtual_mac'],
             prefix_len=self.DEFAULT_PREFIX_LEN,
+            ng_service_enabled=ng_service,
```

Comment:
`ng_service_enabled` True is not tested in `test_ipv6_solicitation`. Does a test make sense here? What is expected if it is True, but `classifier_controller_id` is not set?

Reply:
The ```classifier_controller_id``` value set to 5 in pipelined.yml.
This value used for 5G RYU controller set value for packetIN events.